README.md

```diff
@@ -29,25 +29,31 @@ Bump tries to identify your file specification and format automatically. You can
 * `asyncapi/v2/json`
 * `asyncapi/v2/yaml`
 
-The doc `id` and `token` options used below can be found in your documentation settings page on https://bump.sh. Note that you can replace these options by environments variables: `--id` can be replaced by `BUMP_ID`, and `--token` can by replaced by `BUMP_TOKEN`. This is useful to keep your private token secret.
+`doc` and `token` options used below can be found in your documentation settings page on https://bump.sh. Note that you can replace the token option by an environment variable, to keep it secret: `--token` can by replaced by `BUMP_TOKEN`.
 
 ### Preview
 
 You can preview your documentation by calling the `preview` command. A temporary preview will be created, with a unique URL. This preview will be available for 30 minutes. You don't need any credentials to use this command.
 
+Preview a documentation:
+
     $ bundle exec bump preview path/to/your/file.yml
 
 ### Validate
 
-Validate your file against its specification.
+Validate your file against its specification:
 
-    $ bundle exec bump validate path/to/your/file.yml --id DOC_ID --token DOC_TOKEN
+    $ bundle exec bump validate path/to/your/file.yml --doc DOC_ID_OR_SLUG --token DOC_TOKEN
 
 ### Deploy
 
-Deploy the file as the current version of the documentation.
+Deploy the file as the current version of the documentation:
+
+    $ bundle exec bump deploy path/to/your/file.yml --doc DOC_ID_OR_SLUG --token DOC_TOKEN
+
+Automatically create a documentation inside a hub and deploy it:
```

Comment:
Mmmhh, I don't think these are typos. It uses the infinitive form to describe what the user wants to do, not what the action does. This is the same form we use for other parts of the README, like "Preview a documentation" (instead of "Previews a documentation").

Are you OK with that?

Reply:
Yes you right, misread it!